apps/forms/models.py

```diff
@@ -0,0 +1,155 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.admin.edit_handlers import FieldRowPanel
+from wagtail.admin.edit_handlers import MultiFieldPanel
+from wagtail.admin.edit_handlers import ObjectList
+from wagtail.admin.edit_handlers import TabbedInterface
+from wagtail.contrib.forms.models import AbstractEmailForm
+from wagtail.contrib.forms.models import AbstractFormField
+
+from apps.contrib.translations import TranslatedField
+from apps.settings import helpers
+
+
+class FormField(AbstractFormField):
+    page = ParentalKey('FormPage',
+                       on_delete=models.CASCADE,
+                       related_name='form_fields')
+
+
+class FormPage(AbstractEmailForm):
+    header_de = models.CharField(
```

Comment:
more a general question, but shouldn't we always require the German fields? At least the ones we need to make the page work.

Reply:
Oh yes, true, I changed that to be on its own page, as thats what the designs propose and and empty page would not make sense.. I'll change it to be required then!